README.md

```diff
@@ -40,18 +40,20 @@ MNELAB comes with the following features that are not (yet) available in MNE:
 The latest release is available on [PyPI](https://pypi.python.org/pypi) and can be installed with:
 ```
 pip install mnelab
+mnelab
 ```
-The `mnelab` command starts the application.
+The `mnelab` command in the last line starts the application.
 
 #### Via conda
-An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/), ideally into
-its own dedicated environment:
+An (unofficial, but regularly updated) conda package can be installed from [conda-forge](https://conda-forge.org/).
+We **strongly** suggest to install `mnelab` into its own dedicated environment to ensure smooth installation
```

Comment:
One last nitpick, I'd write MNELAB to refer to the app, `mnelab` is just the command to start it.

Reply:
Done in 503e503